ui/app/AppLayouts/Chat/CommunityComponents/CommunityProfilePopupOverview.qml

```diff
@@ -102,7 +102,7 @@ Column {
         components: [
             StatusSwitch {
                 checked: !chatsModel.communities.activeCommunity.muted
-                onClicked: root.notificationsButtonClicked(checked)
+                onClicked: root.notificationsButtonClicked(!checked)
```

Comment:
for switches, it might be better to handle `onCheckedChanged`, in this case you don't have to guess on `checked` vs `!checked`.

Reply:
@pinebit I *think* tried the option you suggested:
```
Component.onCompleted: {
    checked = !chatsModel.communities.activeCommunity.muted
}
onCheckedChanged: root.notificationsButtonClicked(!checked)
```
I still need the `!checked` to send the correct new value to nim

Is it what you are suggesting, or there is another way?

Reply:
I think `onCheckedChanged` is a good idea too.
What we also usually do is check that the current value is different. Something like:
```qml
                checked: !chatsModel.communities.activeCommunity.muted
                onCheckedChanged: {
                    if (chatsModel.communities.activeCommunity.muted !== !checked) {
                         root.notificationsButtonClicked(!checked)
                    }
                }
```

Reply:
@jrainville assigning checked as you suggest cause a binding error.
Also since latest update on master I am not able to initialize the value with Component.onCompleted